paper/paper.md

```diff
@@ -0,0 +1,135 @@
+the simulation protocols followed. As a result, this wrapper is likely unusable
+for the next project where one of the aforementioned choices changes, and the
+process of designing a workflow must begin again from scratch.
+
+Software packages such as Radonpy [@radonpy_2022] exist that provide an automated workflow for
+building molecules and bulk structures to calculating physical properties of polymers.
+However, these tools may not be suitable for modeling complex experimental
+processes that extend beyond measuring material properties, such as
+simulating fusion welding of polymer interfaces
+[@aggarwal_molecular_2020, @bukowski_load-bearing_2021] and surface wetting
+[@fan_wetting_1995, bamane_wetting_2021].
+
+`flowerMD` is a Python package that consolidates and automates
+end-to-end workflows for modeling such processes with a focus on organic molecules.
+Following the principals of Transparent, Reproducible, Usable by others, and Extensible (TRUE) [@TRUE_2020]
+software design, the modular design of `flowerMD` facilitates building and
+running workflows for specific materials science research applications,
+while reducing the cognitive load and programming demands on the user's part.
+
+# Building Blocks
+`flowerMD` is extensible. Modular base classes serve as building blocks that lay the
+foundation for constructing segregated workflow recipes designed for specific applications.
+The resultant recipes are agnostic to choices such as chemistry, model resolution
+(e.g. atomistic vs. coarse grained) and forcefield selection.
+This is accomplished via three base classes:
+
+• The `Molecule` class utilizes the mBuild [@mbuild_2016] and GMSO [@gmso] packages to initialize chemical
+structures from a variety of input formats. This class provides methods
+for building polymers and copolymer structures, and supports a straightforward
+coarse-graining process by leveraging SMARTS matching.
+
+• The `System` class serves as an intermediary between molecular initialization
+and simulation setup. This class builds the initial configuration and
+applies a chosen forcefield that defines particle interactions.
+
+• The `Simulation` class adds a layer on top of the HOOMD-blue simulation object,
+adding additional methods and features to simplify the process of starting and
+resuming a HOOMD-blue simulation.
+
+Additionally, `flowerMD` offers a library pre-defined subclasses of these base classes
+including common polymers, forcefields, and bulk system initialization algorithms.
+
+# Recipes
+`flowerMD` offers the following two ready-to-go recipes to illustrate how the design creates
+potential for expanding the library of open-source and version-controlled workflows. The included example
+code demonstrates how the modularity of `flowerMD` allows use and re-use of workflows, using
+ pre-built recipe subclasses included in the repository: `SlabSimulation`, `WeldSimulation`, and `Tensile`.
```

Comment:
I think we should include and briefly summarize `Interface` here.  Part of the point of having modules is because some workflows require multiple steps of build a system --> Simulation --> build a system --> simulation where the intermediate system building is unique to the workflow (e.g. Building an interface of 2 slabs from the output of the `SlabSimulation`)

Reply:
See my comment below related tot his.